config/locales/ui.en-GB.yml

```diff
@@ -597,7 +597,7 @@ en-GB:
             </div>
           </details>
       created_by: "Created by"
-      date_closed: "Considered from"
+      date_opened: "Considered from"
```

Comment:
I'd probably go with `date_referred` as the name to indicate the usage - if they were to go back to only considering after closing then the key name still works